docusaurus/src/helpers/objects.js

```diff
@@ -0,0 +1,115 @@
+const get = require("lodash/get");
+
+/**
+ * Generate all possible combinations of a string with options
+ *
+ * Example:
+ * generateCombinations("a_[foo|bar]") // ['a_foo', 'a_bar']
+ *
+ * @param {string} str - The string to generate the combinations from
+ * @returns {string[]} All possible combinations of the string with options
+ */
+const generateCombinations = (str) => {
+  const regex = /\[([^\[\]]+)\]/;
+  let match = str.match(regex);
+
+  if (!match) {
+    return [str];
+  }
+
+  const options = match[1];
+
+  // Check if options are integers
+  // If so, return the original string
+  // This is to maintain compatibility with arrays
+  if (/^\d+$/.test(options)) {
+    return [str];
+  }
+
+  const splitOptions = options.split("|");
+  const prefix = str.substring(0, match.index);
+  const suffix = str.substring(match.index + match[0].length);
```

Comment:
I think youre right. Im not going to implement this now. Since its working and I desperately want to get back to some platform code.

LMK if this NIT, becomes a DO

Reply:
Nope this isn't blocking, just wanted to comment it since I noticed it